docs/resources/docker_container.md.erb

```diff
@@ -39,55 +43,57 @@ Alternatively, you can pass in the container id:
 
 <br>
 
-## Examples
+## Property Examples
 
 The following examples show how to use this InSpec resource.
 
-### Verify an running container:
-
-    describe docker_container('an-echo-server') do
-      it { should exist }
-      it { should be_running }
-      its('id') { should_not eq '' }
-      its('image') { should eq 'busybox:latest' }
-      its('repo') { should eq 'busybox' }
-      its('tag') { should eq 'latest' }
-      its('ports') { should eq [] }
-      its('command') { should eq 'nc -ll -p 1234 -e /bin/cat' }
-    end
-
-<br>
-
-## Matchers
-
-For a full list of available matchers please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/). 
-
 ### id
 
-The `id` matcher tests the container id:
+The `id` property tests the container id:
 
     its('id') { should eq 'sha:71b5df59...442b' }
 
 ### repo
 
-The `repo` matcher tests the value of the image repository:
+The `repo` property tests the value of the image repository:
 
     its('repo') { should eq 'busybox' }
 
 ### tag
 
-The `tag` matcher tests the value of the image tag:
+The `tag` property tests the value of the image tag:
 
     its('tag') { should eq 'latest' }
 
 ### ports
 
-The `ports` matcher tests the value the docker ports:
+The `ports` property tests the value the docker ports:
 
     its('ports') { should eq '0.0.0.0:1234->1234/tcp' }
 
 ### command
 
-The `command` matcher tests the value of the container run command:
+The `command` property tests the value of the container run command:
 
     its('command') { should eq 'nc -ll -p 1234 -e /bin/cat' }
+
+
+### Verify a running container:
+
+    describe docker_container('an-echo-server') do
+      it { should exist }
+      it { should be_running }
+      its('id') { should_not eq '' }
```

Comment:
Future PR can potentially change these to use `cmp` instead of `eq` if we're going to standardize on that. But this is technically correct as-is so I'm going to move it forward.